clang/test/CIR/CodeGen/tbaa.c

```diff
@@ -0,0 +1,121 @@
+// modified from clang/test/CodeGen/tbaa.c
+// with clangir enabled and some debugging info
+// if disable debuggging info is supported by clangir
+// the debugging info can be removed
+// and keep the test case the same with clang/test/CodeGen/tbaa.c
+
+// RUN: %clang_cc1 -triple x86_64-unknown-linux-gnu -fclangir -O1 -no-struct-path-tbaa -disable-llvm-passes %s -emit-llvm -o - | FileCheck %s
```

Comment:
Few additional nits:
- Please make all tests are uniform w.r.t to FileCheck prefixes (similar to what you did in `clang/test/CIR/CodeGen/tbaa3.c`)
- Different RUN lines for FileCheck invocations (it's usually fine to pipe directly but since this is initial support, it makes it way more convenient to review when running tests and clicking at an output file, instead of having to do that by hand when trying to look at the full output).

Example for this specific case: `LLVM_NO_STRUCT` and `LLVM_STRUCT`